### PR TITLE
[feature] docker build integration

### PR DESCRIPTION
**Pull request by sescobb27**

### CODE REVIEW

File: apps/domain/lib/tasks/task.ex [hunk 1]
```diff
@@ -28,17 +28,40 @@ defmodule Domain.Tasks.Task do
...
 
-  defstruct url: nil, path: nil, env: [], commands: [], runner: Make, source: Github
+  defstruct url: nil,
+            build_file: nil,
+            build_file_content: [],
+            path: nil,
+            env: [],
+            commands: [],
+            runner: Make,
+            source: Github
```
Comment (ghost) [inline apps/domain/lib/tasks/task.ex:48]: Very nice - makes the interface very clear and retains information after it's actualized.

**Review by ghost — APPROVED** [review]

**Comment by sescobb27** [comment]
rebased with master

File: .travis.yml [hunk 2]
```diff
@@ -1,21 +1,23 @@
 language: elixir
 install: true
 sudo: false
-before_install: 
+before_install:
```
Comment (sescobb27) [inline .travis.yml:4]: ups, sorry, my editor trims whitespaces automatically

**Comment by ghost** [comment]
LGTM

**Pull request merged**